apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -100,16 +100,19 @@ public sealed class CloudTrace : IDisposable
         /// </example>
         /// </summary>
         public static TraceHeaderPropagatingHandler CreateTracingHttpMessageHandler() =>
-            new TraceHeaderPropagatingHandler(() => CurrentTracer);
+            new TraceHeaderPropagatingHandler(() => Tracer);
```

Comment:
Presumably at this point you could actually change TraceHeaderPropagatingHeader to accept just an IManagedTracer, and pass in a delegating one everywhere it's used? No particular need, just an option... it may be clearer to keep it the way it is.